### PR TITLE
Small tweaks

### PR DESCRIPTION
**Pull request by doublebyte1**
- Fixed formatting issues
- Added link and image to standards section
- Removed advanced empty sub-sections

### CODE REVIEW

**Comment by tomkralidis** [comment]
thanks @doublebyte1!

@francbartoli are you going to add anything about custom plugins or security/access control?  They were originally included as TODOs, but if we don't include anything by Florence, suggest we delete (per this PR) and add them to the issue tracker for future work.

File: workshop/content/docs/advanced/index.md [hunk 1]
```diff
@@ -9,9 +9,7 @@ focused on extending pygeoapi via custom development and deployment.
 
 - [Multilingual support](i18n.md)
 - [UI customization and templating](ui-custom-templates.md)
-- [Custom plugins](custom-plugins.md)
 - [Using pygeoapi in downstream applications](downstream-applications.md)
-- [Security and access control](security-access-control.md)
```
Comment (francbartoli) [inline workshop/content/docs/advanced/index.md:14]: We can keep this line and the markdown file

**Review by francbartoli — CHANGES_REQUESTED** [review]

**Comment by francbartoli** [comment]
@tomkralidis I'm going to add only security and access control today. PR is coming later

**Review by tomkralidis — CHANGES_REQUESTED** [review]
@doublebyte1 can you put back the security deleted Markdown file (see Francesco's comment)? I can then merge. Thanks

**Comment by doublebyte1** [comment]
https://github.com/geopython/diving-into-pygeoapi/commit/117f73a66f506624ec791c98c4b5c2c3428d6aa1

https://github.com/geopython/diving-into-pygeoapi/commit/8f566642b66eb04b937162883f068a4f1bdb7fb5

Does this work?

**Comment by tomkralidis** [comment]
Thanks @doublebyte1 !

**Pull request merged**